packages/sdk/src/types/DataBlob.ts

```diff
@@ -30,10 +33,15 @@ export class DataBlob {
         this.data = Buffer.from(data);
     }
 
-    public toJSON(): string {
+    public toJSON(): HexString {
         return packBufferWithWord16Length(this.data).toString('hex');
     }
 
+    public static fromJSON(value: HexString): DataBlob {
+        // The first 2 bytes are the length of the data buffer, so we need to remove them.
```

Comment:
Would be good to have this comment as a doc-comment for the function instead to provide documentation for users. Would make sense to describe that it parses the format of `DataBlob.toJSON`